multi-node/aws/pkg/config/config.go

```diff
@@ -96,12 +94,18 @@ type Cluster struct {
 
 func (c Cluster) Config() (*Config, error) {
```

Comment:
All configuration values computed from Cluster parameters are derived/set in this method.